src/libraries/Common/src/Roslyn/SyntaxValueProvider_ForAttributeWithMetadataName.cs

```diff
@@ -0,0 +1,188 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This is one of two files containing the meat of the change.  the entire incremental algorithm, with tweaks to fit into the runtime.